internal/impl/postgresql/pglogicalstream/sanitize/sanitize.go

```diff
@@ -363,28 +363,72 @@ func SQLQuery(sql string, args ...any) (string, error) {
 	return query.Sanitize(args...)
 }
 
-// ValidatePostgresIdentifier checks if a string is a valid PostgreSQL identifier
+// QuotePostgresIdentifier returns the valid escaped identifier.
+func QuotePostgresIdentifier(name string) string {
```

Comment:
```go
return `"` + name.Replace(`"`, `""`) + `"`
```
?

Reply:
Yeah that's simpler. This happens to be faster, but this isn't on a critical path. I'll fix it next time I hack on this input